Tests/AK/TestArray.cpp

```diff
@@ -20,11 +20,29 @@ static constexpr int constexpr_sum(ReadonlySpan<int> const span)
 TEST_CASE(compile_time_constructible)
 {
     constexpr Array<int, 4> array = { 0, 1, 2, 3 };
-    static_assert(array.size() == 4);
```

Comment:
Why replay static_assert with an EXPECT_EQ? checking at compile time > checking at run time, no?

Reply:
It didn't even occur to me that it was intentional because it's compile time, I just assumed it pre-dated the EXPECT macros. That makes sense though.